### ISSUE TITLE
⚡ Reviews & Feedback

### ISSUE THREAD

**Issue by Evavic44** [Evavic44/rocketmeme#35]
## Rocketmeme | Create, Download, and Share Memes

Please drop your reviews and feedback for this project below.

[http://localhost:3000/contact](http://localhost:3000/contact)
[https://rocketmeme.com/contact](https://rocketmeme.com/contact)
[https://rocketmeme.netlify.app/contact](https://rocketmeme.netlify.app/contact)
👇🏽👇🏽👇🏽👇🏽👇🏽👇🏽👇🏽👇🏽👇🏽👇🏽

**Comment by Evavic44** [comment]
Just a cool test. 😅

**Comment by Evavic44** [comment]
Sign in and drop your feedback below. 🧡😅💛

**Comment by israelmitolu** [comment]
I really love the UI and the app.

However, I had issues signing up with Google

**Comment by Evavic44** [comment]
Yes, you're right Israel. We're currently working on a fix for that. 😅

**Comment by Evavic44** [comment]
Hello World
